ext/redhat/facter.spec.erb

```diff
@@ -94,11 +94,11 @@ rm -rf %{buildroot}
 
 %files
 %defattr(-,root,root,-)
-%{_bindir}/cfacter
+%{_bindir}/facter
 %{_libdir}/libfacter.so*
 %{_includedir}/facter
 %if 0%{?rhel} >= 7 || 0%{?fedora}
-%{vendor_ruby}/cfacter.rb
+%{vendor_ruby}/facter.rb
```

Comment:
Oh, got it. Do we treat Facter 3 as a drop-in replacement for Facter 2 against Puppet 4.0? Not sure either, I don't have a strong reason for doing it.